app/cypress/e2e/0-ndr-core-tests/gp_user_workflows/download_lloyd_george_workflow.cy.js

```diff
@@ -59,10 +59,17 @@ describe('GP Workflow: View Lloyd George record', () => {
             body: testFiles,
         }).as('searchDocumentReferences');
 
+        cy.intercept('POST', '/DocumentManifest*', (req) => {
+            req.reply({
+                statusCode: 200,
+                body: { jobId: 'test-jobId' },
+            });
+        }).as('documentManifestPost');
+
         cy.intercept('GET', '/DocumentManifest*', (req) => {
             req.reply({
                 statusCode: 200,
-                body: baseUrl + '/browserconfig.xml', // uses public served file in place of a ZIP file
+                body: { jobStatus: 'Completed', url: baseUrl + '/browserconfig.xml' }, // uses public served file in place of a ZIP file
```

Comment:
Could you add a test case for if the jobStatus is still pending and the url is empty?

Reply:
Thanks, I have added the pending to the base test case. The intercept will now respond with "Processing" and an empty url for a few times before it responds with "Complete"

Reply:
Also add a test case for getting "Pending" 3 times and display an error page